manifests/candlepin.pp

```diff
@@ -41,6 +41,27 @@
     password_file => $::certs::ca_key_password_file,
   }
 
+  $tomcat_cert_name = "${hostname}-tomcat"
+  $tomcat_cert = "${certs::pki_dir}/certs/katello-tomcat.crt"
```

Comment:
Please use ```${::certs::pki_dir}``` or even better: fix the class parameters to always use ```$::certs::*``` instead of ```$::certs::params::*``` so you can just use ```$pki_dir```.